includes/discovery/mempools/fortiweb.inc.php

```diff
@@ -1,8 +1,11 @@
 <?php
-echo 'FORTIWEB-MEMORY-POOL: ';
-$usage = str_replace('"', "", snmp_get($device, 'FORTINET-FORTIWEB-MIB::fwSysMemUsage.0', '-OvQ'));
-if (is_numeric($usage)) {
-    // fwSysMemCapacity: Total physical memory (RAM) installed (MB)
-    // Conversion from megabyte (base 10) to byte (base 10): Precision = 1000*1000
-    discover_mempool($valid_mempool, $device, 0, 'fortiweb', 'Physical Memory', '1000000', null, null);
+
+if ($device['os'] == 'fortiweb') {
```

Comment:
These checks are not needed anymore right?

Reply:
they are. All files are loaded without condition right now. So we end up testing fortiweb on all devices for each discovery :)

Reply:
yes, the mempools code just loops over all the files... as I said very old code.